src/pytest/__main__.py

```diff
@@ -0,0 +1,6 @@
+"""
+pytest entry point
+"""
+import pytest
+
```

Comment:
i vaguely recall its necessary to check for `__name__ == "__main__"` to guard against certain user mistakes

Reply:
OK, will update it later, thanks